test/test_optim.py

```diff
@@ -1368,51 +1378,55 @@ def fwd_bwd(optim, w, b, i):
             return loss
 
         for optim_input in all_optim_inputs:
-            optimizer = optim_cls(params, **optim_input.kwargs)
-            closure = functools.partial(fwd_bwd, optimizer, weight, bias, input)
-
-            # Prime the optimizer
-            for _ in range(10):
-                if optim_info.step_requires_closure:
-                    optimizer.step(closure)
-                else:
-                    closure()
-                    optimizer.step()
+            for param_in in make_param_and_named_param(params):
```

Comment:
Stylistic nit: Instead of parametrizing in here, could you add this as a parameterization to the test (so there are two distinct tests vs one large test)?